worker/pkg/benthos/sql/processor_neosync_mssql.go

```diff
@@ -0,0 +1,139 @@
+package neosync_benthos_sql
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/nucleuscloud/neosync/internal/gotypeutil"
+	neosync_benthos "github.com/nucleuscloud/neosync/worker/pkg/benthos"
+	"github.com/warpstreamlabs/bento/public/service"
+)
+
+func neosyncToMssqlProcessorConfig() *service.ConfigSpec {
```

Comment:
Would be nice to see tests for these things via the benthos-way of testing them where they are init'd and run through that, more like an integration test.

See the out `output_sql_insert_test.go` for a basic example of what I'm referring to..